docs/developers/contributing-guidelines.md

```diff
@@ -0,0 +1,142 @@
+# Code contributions
+
+Project and code leads are experienced HOT volunteer and staff developers and the main points of contact for the project. They are also the final reviewers of issues and pull requests. Code leads will review pull requests and provide feedback. The purpose of this role is to help contributors,
```

Comment:
To explain here HOT_tech team role and KLL as maintainers. HOT_tech merges to masters, whereas KLL merges to develop and other developers/volunteers can contribute. https://github.com/hotosm/tasking-manager/pull/5116